### DISCUSSION TITLE
cdc 通过保存点重启任务, 在yaml文件中新增的表不会进行同步该怎么办?

### DISCUSSION THREAD

**Discussion by 1154845240** [apache/flink-cdc#3690] [Q&A]
通过保存点保存之后, 在yaml文件新增三张表, 通过保存点重启新增的三张表并未同步
![image](https://github.com/user-attachments/assets/08856c67-1719-4b07-a44e-0f7e2cdbf2c4)
energo_saas_2.sys_equipment_error_record,energo_saas_2.sys_equipment_error_type,energo_saas_2.sys_equipment_switch为新增的表,

**Comment by beryllw** [comment]
Try using the MySQL source option `scan.newly-added-table.enabled`.
https://nightlies.apache.org/flink/flink-cdc-docs-release-3.2/docs/connectors/pipeline-connectors/mysql/

**Comment by namedai** [comment]
请问如何通过保存点重启任务的
